cbmc/proofs/DeserializeConnack/DeserializeConnack_harness.c

```diff
@@ -0,0 +1,34 @@
+/* Build using a config header, if provided. */
+#ifdef AWS_IOT_CONFIG_FILE
+    #include AWS_IOT_CONFIG_FILE
+#endif
+
+/* Standard includes. */
+#include <string.h>
+
+/* MQTT internal includes. */
+#include "private/aws_iot_mqtt_internal.h"
+
+AwsIotMqttError_t
+AwsIotMqttInternal_DeserializeConnack( const uint8_t * const pConnackStart,
+				       size_t dataLength,
+				       size_t * const pBytesProcessed );
+
+void harness()
+{
+  // Buffer can be any buffer of size at most BUFFER_SIZE
+  uint8_t buffer[BUFFER_SIZE];
+  size_t buffer_offset;
+  size_t buffer_size = BUFFER_SIZE - buffer_offset;
+
+  __CPROVER_assume(buffer_offset <= BUFFER_SIZE);
```

Comment:
Nit picking about space: the next harness does not have a blank line before the `__CPROVER_assume`. Either is fine, but consistency would be nice to have.

Reply:
This is gone in the squashed commit.